include/config/config.h.in

```diff
@@ -48,6 +48,7 @@
 #cmakedefine WITH_OPENCL
 #cmakedefine WITH_MEDIA_FOUNDATION
 #cmakedefine WITH_MEDIACODEC
+#cmakedefine WITH_V4L
```

Comment:
I´d suggest using `add_definitions` in `channels/rdpecam/client` for that.
this way you can move the `libv4l` detection to the channel too and do not need to touch code outside the channel

Reply:
All of it is done, as suggested